scripts/0_wipe.sql

```diff
@@ -0,0 +1 @@
+DROP DATABASE %s;
```

Comment:
nit: There's almost never something broken by this but your editor seems to consistently leave off the trailing new line. That's what these little markings mean.